bids/README.md

```diff
@@ -23,41 +25,138 @@ The neurospin_to_bids.py script can be use without install the unicog module.
 The test_dataset follows a minimal set of conventions for the download to work.
 
 
-# Using:
+# Basic usage:
 
-        cd path_to/test_dataset
-        python path_to/neurospin_to_bids.py -neurospin_database trio
+        cd <path_to/test_dataset>
+        python <path_to>/neurospin_to_bids.py -neurospin_database trio
 
+* The **\<path_to/test_dataset\>** directory contains an **exp_info** directory with 2 files 
+(**participants.tsv** and **download.tsv**) indicating what are the subjects to download.
+For details on the files content/structure, read the Additional information section.
+* The **neurospin_database** parameter refers to the neurospin server where your 
+data are stored, either "prisma" or "trio" (prisma is the default parameter).
+
+# Advanced usage:
+
+        python neurospin_to_bids.py -root_path <root_path> -dataset_name <my_dataset> -neurospin_database trio
+
+* The **root_path** option allows to indicate a specific path including the **exp_info** directory.
+* The **dataset_name** option allows to give a specific name to the dataset (the default is bids_dataset).
 
 # Additionnal information
-## Input files for the preparation of importation:
-Two input files are needed:
-- participants.tsv: corresponds to the list of participants. Here is an example:
+## Input files for the importation of data:
+Two files are mandatories: **participants.tsv** and **[sub-*_][ses-*_]download.csv**. Their configuration is depending
+to the dataset (number of sessions, runs, tasks ....). Many cases are presented below.
+
+### One session/subject 
+- **participants.tsv**: corresponds to the list of participants. Some additional field can be added such
+as sex, age ... 
 
         participant_id	NIP	acq_date
-        sub-1	nip_number	2010-06-28
-        sub-2	nip_number	2010-07-01
+        sub-01	nip_number	year-month-day
+        sub-02	nip_number	year-month-day
+
+- **download.tsv**: corresponds to the acquisitions to download.
+If the number of acquisitions is the same for all subjects, you have to indicate only 
+the acquisition number once. Here is an example:
```

Comment:
In the second example you include runs and not here, so I think this can be confusing for someone that does not know the details of BIDS if we dont provide clarification